store/searchtest/file_info_layer.go

```diff
@@ -1214,7 +1256,7 @@ func testFileInfoSearchDiscardWildcardAlone(t *testing.T, th *SearchTestHelper)
 
 	p1, err := th.createFileInfo(th.User.Id, post.Id, "qwerty", "qwerty", "jpg", "image/jpeg", 0, 0)
 	require.NoError(t, err)
-	_, err = th.createFileInfo(th.User.Id, post.Id, "qwertyjkl", "qwertyjkl", "jpg", "image/jpeg", 0, 0)
+	_, err = th.createFileInfo(th.User.Id, post.Id, "werty", "werty", "jpg", "image/jpeg", 0, 0)
```

Comment:
I changed this part to match the code change

Reply:
The purpose of this test is to show that wildcards cannot be used alone.
` "qwerty *"` is used as a term for testing but the test failed because the wildcard was added to every terms.
(`qwerty* matches qwertyjkl)